src/core/Pricing/data.tsx

```diff
@@ -229,27 +229,24 @@ export const consumptionData: PricingDataFeature[] = [
       color: "text-neutral-000",
       tooltip: (
         <p>
-          Ably aggregates all its data into named units of distribution,
-          referred to as “channels”. These are used to transmit from one device
-          to another.{" "}
-          <a href="/docs/channels" className="ui-link">
-            Find out more.
-          </a>
+          We charge you for the amount of time a channel is active in our
+          network by minute. For example, if ten channels are in use for 45
```

Comment:
Shouldn't this be "in our network by _THE_ minute"? Doesn't read well (to me at least)